cmd/tailscale/cli/serve.go

```diff
@@ -30,9 +30,9 @@ import (
 	"tailscale.com/version"
 )
 
-var serveCmd = newServeCommand(&serveEnv{})
+var serveCmd = newServeCommand(&serveEnv{lc: &localClient})
```

Comment:
why the `&`?

The variable is of type:

```
var localClient *tailscale.LocalClient
```

And those three methods are already defined on the LocalClient pointer.

Reply:
I see the variable type as the struct:
https://github.com/tailscale/tailscale/blob/6edf357b96b28ee1be659a70232c0135b2ffedfd/cmd/tailscale/cli/cli.go#L127

Reply:
Ah I was looking in a different package.